tasks/18_humanize-format/humanize-format.js

```diff
@@ -0,0 +1,18 @@
+export const humanizeFormat = (...args) => {
```

Comment:
I used rest operator because of the first test case - `expect(humanizeFormat()).toBe('')`. Since this case doesn't include any argument, I checked `args.length === 0` to see if there is an argument passed to the function.